### ISSUE TITLE
Beta layer: Projected Climate -- official CLIMAID Projections

### ISSUE THREAD

**Issue by colinbeier** [caryfitzhugh/NYCCSC-map-viewer#59]
NYSERDA wants the official CLIMAID projections in the Clearinghouse.  Art has them, but does not plan on incorporating them in ACIS.  They are coarse-resolution (0.25 degree) grids, not downscaled, which limits their utility in a mapping space.  

We need to figure out how to deal with these data.  We are going to have to serve them ourselves.  Art & Bill are suggesting using geojson, but I'm not sure that's a good idea in the long run.  Also concerned that if our map is the ONLY place on the site where the CLIMAID projections can be found, then the expectation of being able to do more than just visualize the data image will develop.

I tentatively told Art to have Bill generate the jsons so that we could use them as placeholders for the beta (and potentially final layers?), but I will follow whatever approach you guys think is best.  

This is one of several jsons or ACIS query calls that Bill has agree to provide us.  I presume Bill will contact both Steve and I when he does this.  We should touch base with Bill if we don't hear something by EOB Wed.

**Comment by sasignell** [comment]
Json is fine...I can convert to whatever we need

**Comment by colinbeier** [comment]
OK, we're back to json format.  Art indicated earlier that he would not be providing geojson, but Steve asked already, so we'll see what he says.  

Either way -- there will be two layers: projected changes in mean annual temp (MAT) and total annual precip (TAP).  Each will be provided as a separate dataset.  The data is clipped to a bounding box.  

Within each layer there are three nested variables: 
> Year (3 values: 2020, 2050, 2080)
>> Scenario (2 values: 85 = high emissions, 45 = low emissions)
>>> Model ensemble percentile (10, 25, 50 (median), 75, 90)  

This equates to 30 sub layers for MAT and TAP projections, each (!!)... so I would really like to trim this down, but need guidance from NYSERDA and Content Team before we can make final decisions.  If some trimming would be helpful now for the beta, then we can remove the 2020 date, which gets us down to 20 sub layers (a reduction by 1/3). 

The actual data values are the deltas, or departures, from the model simulated historical period (1980-2010).  As noted above the grids are coarse resolution, not the best for mapping, but NYSERDA really wants them in the site.  

Once we get the MAT data in a format we like, then we can get the TAP data from Art. 

Let me know if you have questions.

**Comment by caryfitzhugh** [comment]
is this all in?

**Comment by colinbeier** [comment]
for the BETA, yes.  not for final.  i'm sure we'll circle back around to this, but i will start a new issue then.
